sweetalert2.d.ts

```diff
@@ -550,7 +550,7 @@ declare module 'sweetalert2' {
          *
          * @default null
          */
-        preConfirm?: (inputValue: any) => SyncOrAsync<any | void>;
+        preConfirm?: (inputValue: string) => SyncOrAsync<any | void>;
```

Comment:
This one is wrong: preConfirm can also take a boolean (it's actually the classic case, when there is no input).
I can find only one place where SweetAlert2 returns another thing than a string input value: https://github.com/sweetalert2/sweetalert2/blob/2b33d1906b5ec1c9963aa7540637f231398c782e/src/sweetalert2.js#L258

We can infer that the correct type is actually `true | string`.

But I don't like it. What if, one day, preConfirm can receive other data types? I propose to keep `any` for now.

Reply:
Done. I've removed this change. Take a look please :)

Reply:
@toverux ping :)